frontend/src/components/LoadingButton.tsx

```diff
@@ -0,0 +1,11 @@
+import { Loader2 } from 'lucide-react';
+import { Button } from './ui/button';
+
+const LoadingPageButton = () => {
+  return (
+    <Button disabled>
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/19#issuecomment-2379954551>#1588a1</a><span> - 09/27/2024, 08:00 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The LoadingPageButton component lacks proper accessibility attributes for screen readers, specifically for indicating the loading state.
</div>
<div id="fix">
<b>Fix:</b> Add appropriate ARIA attributes to the Button component to indicate its loading state and purpose.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -4,7 +4,7 @@
  const LoadingPageButton = () => {
    return (
 -    <Button disabled>
 +    <Button disabled aria-busy="true" aria-label="Loading, please wait">
        <Loader2 className="mr-2 h-4 w-4 animate-spin">Loading</Loader2>
      </Button>
    );
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged